resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -160,7 +160,7 @@ object DbReader {
               NOT EXISTS (
                 SELECT *
                 FROM NODEPOOL np
-                LEFT JOIN APP a ON np.id = a.nodepoolId
+                RIGHT JOIN APP a ON np.id = a.nodepoolId
                 WHERE
                   kc.id = np.clusterId AND np.isDefault = 0 AND
                   (
```

Comment:
Can probably remove `OR (a.id IS NULL)` at the end because that's impossible with a right join, correct?

Reply:
I'm not super sure... currently, this query matches cluster with nodepools and no apps. Not familiar enough with SQL to know if `a.id` would be null in that scenario.